README.md

```diff
@@ -12,20 +12,20 @@ read, reason about, and test.
 ## Why Reginn?
 
 Reginn was built to eliminate the crutch on monolithic imperative APIs to work
-with the command line.
+with on the command line.
 
 Leading alternatives are both largely unmaintained and
-feature large imperative classes, which are bad for reusability. Many of them
-introduces many new syntaxes that you have to learn and all are based around
-the way you format strings. As you know, that is a recipe for disaster.
+feature large imperative classes, which tend to be bad for reusability. Many of them
+introduce lots of new syntax that you have to learn and they all are based around
+the way you format strings. That is usually a recipe for disaster.
 Many alternatives also force the intermixing of where you declare your application’s
 intent and where side effects exist that leads to poor separation of concerns
 and testability.
 
 Reginn is laser-focused to create small, testable functions that allow you to
 build you command line applications with assurances not found elsewhere. Reginn
-introduces zero new syntaxes to your application and no new concepts, since you
-already know functions.
+introduces zero new syntax to your application and adds no new concepts, as we are
```

Comment:
I could also argue against the notion of "not adding new concepts" as in fact, Reginn's API is adding new concepts in the form of new types. It could be worded something like this:

> introduces zero new syntax to your application and only adds some lego-like functions that can be composed together to create your application.